src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { renderHook } from '@testing-library/react-hooks';
+
+import axios from 'axios';
+
+import GeneralContextProvider from '../../state/GeneralContext';
+import { useGeneralContext } from '../../state/GeneralContext';
+import Home from './Home.page.jsx';
+import mockResponse from '../../utils/mockApodApiResponse';
+
+const RenderHtml = () => {
+  return(
+    <GeneralContextProvider>
+      <Home></Home>
```

Comment:
Este componente se puede cerrar en la misma etiqueta que lo abre porque no tiene children.
```suggestion
      <Home/>
```